check-alignment.sh

```
@@ -0,0 +1,117 @@
+#!/bin/bash
+progname="${0##*/}"
+progname="${progname%.sh}"
+
+# usage: check_elf_alignment.sh [path to *.so files|path to *.apk]
+
+cleanup_trap() {
+  if [ -n "${tmp}" -a -d "${tmp}" ]; then
```

Shellcheck (0.11.0)

[warning] Prefer [ p ] && [ q ] as [ p -a q ] is not well defined.

(SC2166)